src/core/ddsi/include/dds/ddsi/q_bitset.h

```diff
@@ -13,40 +13,41 @@
 #define NN_BITSET_H
 
 #include <assert.h>
+#include <stdint.h>
 #include <string.h>
 
 #include "dds/ddsi/q_unused.h"
 
-inline int nn_bitset_isset (unsigned numbits, const unsigned *bits, unsigned idx)
+inline int nn_bitset_isset (uint32_t numbits, const uint32_t *bits, uint32_t idx)
 {
   return idx < numbits && (bits[idx/32] & (1u << (31 - (idx%32))));
```

Comment:
The 1u now also needs to be (uint32_t)1: if uint32_t need not be equivalent to unsigned (and in particular, if unsigned can be narrower than uint32_t, using 1u might result in undefined behaviour here). Same holds for of course for all these constants in this file.

Reply:
Switched to using UINTxx_C.